### ISSUE TITLE
text height not working

### ISSUE THREAD

**Issue by pcapanna** [song940/node-escpos#34]
When setting other sizes other than 1x1, 1x2, 2x1 or 2x2, text height is always 1, and width whatever you put on it.

I had to add a function on commands.js.. but it works fine and I didn't have to add 64 lines of literals possibilities.

I changed/added on commands.js:
```
_.TEXT_FORMAT = {
  ...
  TXT_OTHER_SIZE  : function(width, height){ // other sizes
    var charsPreSize = '\x1d\x21';
    var decWidth = (width - 1) * 16;
    var decHeight = height - 1;
    var charPostSize = String.fromCharCode(decWidth+decHeight)
    return charsPreSize + charPostSize;
  },
```
And on printer.js on size function:

```
Printer.prototype.size = function(width, height) {

  if (2 >= width && 2 >= height) {
    ...
  }
  else {
    this.buffer.write(_.TEXT_FORMAT.TXT_OTHER_SIZE(width,height));
  }
```

**Comment by Sophtware** [comment]
I ran into the same issue as the one you described. Thanks for putting up a potential solution. I'll give it a try if I find that I need it. :)

**Comment by pcapanna** [comment]
Why was this issue closed? I can't see the changes in b6ac6b4. I will make a pull request about this so it's easier.

**Comment by pcapanna** [comment]
#43
